pkgs/tools/misc/archi/default.nix

```diff
@@ -7,6 +7,8 @@
 , libsecret
 , webkitgtk
 , wrapGAppsHook
+, archi
```

Comment:
This isn't necessary and in fact will infrec if actually accessed.

Reply:
Thanks! Nice catch, I removed this argument.